occupancy_grid/point_msg[] points
